### ISSUE TITLE
Add post for paper

### ISSUE THREAD

**Issue by beatrizserrano** [usegalaxy-eu/website#1036]
Amir published this: https://www.sciencedirect.com/science/article/pii/S1567134822001150?via%3Dihub

**Comment by beatrizserrano** [comment]
oh and also this, but it's not open: https://www.sciencedirect.com/science/article/pii/S1567134822001150?via%3Dihub

**Comment by erxleben** [comment]
is it not the same? @beatrizserrano

**Comment by beatrizserrano** [comment]
yes, sorry, it's the same! I meant this: https://www.eurekaselect.com/article/123662

**Comment by bgruening** [comment]
this could be: https://github.com/usegalaxy-eu/website/pull/1050/files

**Comment by bgruening** [comment]
https://pubs.acs.org/doi/full/10.1021/acs.jproteome.3c00538

**Comment by bgruening** [comment]
https://www.erga-biodiversity.eu/post/erga-at-the-embo-hands-on-course-in-genome-sequencing-assembly-and-downstream-analyses @erxleben maybe something for you?
